File: src/IsbInterop/Data/IConnection.vb
Imports IsbInterop.Accessory

Namespace Data

  ''' <summary>
  ''' Соединение.
  ''' </summary>
  Public Interface IConnection
    Inherits IIsbComObjectWrapper

    ''' <summary>
    ''' Получает информацию о системе.
    ''' </summary>
    ''' <returns>Информация о системе.</returns>
    Function GetSystemInfo() As ISystemInfo
  End Interface
End Namespace

File: src/IsbInterop/Data/IConnection.cs
using IsbInterop.System;

namespace IsbInterop.Data
{
  /// <summary>
  /// Соединение.
  /// </summary>
  public interface IConnection : IBaseIsbObject
  {
    /// <summary>
    /// Получает информацию о системе.
    /// </summary>
    /// <returns>Информация о системе.</returns>
    ISystemInfo GetSystemInfo();
  }
}
